### DISCUSSION TITLE
Forms return 'false' within stack

### DISCUSSION THREAD

**Discussion by daveada88** [ditdot-dev/vue-flow-form#265] [Q&A]
Hi all,

Can anyone assist with why questions return 'false' within the placeholder when operating within a wider Vue JS stack? I've placed 'Example.vue' (Questionnaire) within a Userfront Authentication environment, which stops operation. The placeholder value displays 'false' and the continue button isn't displayed.

Exactly the same code operates fine when in a standalone environment.

**Comment by spinn** [comment]
Can you give any kind of example, or a repository which we could look at?
